frappe/public/less/quill.less

```diff
@@ -131,13 +131,28 @@
 
 .ql-editor ol {
 	padding-left: 2.5em;
+  padding-right: 2.5em;
 }
 
 .ql-editor li[data-list=ordered] {
 	list-style-type: decimal;
 	padding-left: 0;
+  padding-right: 0;
 
 	&::before {
 		content: none;
 	}
 }
+
+// RTL alignment
+.control-input {
+  direction: ltr;
+}
+
+.ql-container {
+  direction: ltr;
+}
```

Comment:
We can also do something like this:
```
.ql-editor-rtl {
	.ql-editor {
		direction: rtl;
		text-align: right;
	}
}

.ql-editor-ltr {
	.ql-editor {
		direction: ltr;
		text-align: left;
	}
}
```

This way the alignment can be set by default when the language is changed (by adding the class to the editor as per the direction).

Reply:
I just had to change
 `.control-input {
  direction: ltr;
}`
to
`.control-input {
  text-align: left;
}`
and every thing is fine